demos/cms/cms_ver.c

```diff
@@ -27,6 +27,8 @@ int main(int argc, char **argv)
     /* Set up trusted CA certificate store */
 
     st = X509_STORE_new();
+    if (!st)
```

Comment:
Please use `st == NULL` instead.  You can unify the style in the below ifs as well, if you want.

Reply:
done, Thanks for you review.